hanami-context-logging.gemspec

```diff
@@ -22,9 +22,9 @@ Gem::Specification.new do |spec|
   end
 
   # Specify which files should be added to the gem when it is released.
-  # The `git ls-files -z` loads the files in the RubyGem that have been added into git.
+  
   spec.files         = Dir.chdir(File.expand_path('..', __FILE__)) do
-    `git ls-files -z`.split("\x0").reject { |f| f.match(%r{^(test|spec|features)/}) }
+    Dir.glob('**/*').select { |f| File.file?(f) && !f.match(%r{^(test|spec|features)/}) }
```

Comment:
Tested/verified the new command in Hanami Console that in filters out the folders we want.